### PR TITLE
RHDEVDOCS-5795: Update table capitalization

### PR DESCRIPTION
**Pull request by nebelsprachen**
• **Aligned team:** Dev Tools
• **Please cherry-pick to:** `gitops-docs-1.10` `gitops-docs-1.11` `gitops-docs-1.12`

Version(s):

`gitops-docs-1.12`, `gitops-docs-1.11`, `gitops-docs-1.10`  

Issue:

[RHDEVDOCS#5795](https://issues.redhat.com/browse/RHDEVDOCS-5795)

Link to docs preview:

https://72115--docspreview.netlify.app/openshift-gitops/latest/argocd_instance/argo-cd-cr-component-properties

QE review:
- [x] QE has approved this change.

Additional information:

### CODE REVIEW

File: modules/gitops-argo-cd-properties.adoc [hunk 1]
```diff
@@ -9,136 +9,118 @@
...
+  * _<service.Type>_ - The `serviceType` used for the service resource.
+  * _<logLevel>_ - The log level to be used by the Argo CD Server component. Valid options are  `debug`, `info`, `error`, and `warn`.
+  * _<logFormat>_ - The log format used by the Argo CD Application Controller component. Valid options are `text` or `json`.
+  * _<env>_ - Environment to set for the server workloads.
+
+|`sso` |Single Sign-on options.|`__<object>__`
+proa|* _<image>_ - The container image for Keycloak. This overrides the `ARGOCD_KEYCLOAK_IMAGE` environment variable.
+  * _<keycloak>_ - Configuration options for Keycloak SSO provider.
+  * _<dex>_ - Configuration options for Dex SSO provider.
+  * _<provider>_ - The name of the provider used to configure Single Sign-on. For now the supported options are Dex and Keycloak.
```
Comment (skopacz1) [inline modules/gitops-argo-cd-properties.adoc:110]: Small wording suggestion:
```suggestion
  * _<provider>_ - The name of the provider used to configure Single Sign-on. Currently, the supported options are Dex and Keycloak.
```

File: modules/gitops-argo-cd-properties.adoc [hunk 2]
```diff
@@ -9,136 +9,118 @@
...
+  * _<resources>_ - The container compute resources.
+  * _<verifyTLS>_ - Defines whether to enforce strict TLS checking when communicating with Keycloak service.
+  * _<version>_ - The tag to use with the Keycloak container image.
+|`statusBadgeEnabled` |Enable application status badge.|`true` |
+
+|`tls` |TLS configuration options.|`__<object>__`
+a|* _<ca.configMapName>_ - The name of the `ConfigMap` which contains the CA certificate.
+  * _<ca.secretName>_ - The name of the secret which contains the CA Certificate and Key.
+  * _<initialCerts>_ - Initial set of certificates in the `argocd-tls-certs-cm` config map for connecting Git repositories via HTTPS.
```
Comment (skopacz1) [inline modules/gitops-argo-cd-properties.adoc:119]: Per [ISG word usage](https://www.ibm.com/docs/en/ibm-style?topic=word-usage#v), avoid using via:
```suggestion
  * _<initialCerts>_ - Initial set of certificates in the `argocd-tls-certs-cm` config map for connecting Git repositories through HTTPS.
```

File: modules/gitops-argo-cd-properties.adoc [hunk 3]
```diff
@@ -9,136 +9,118 @@
...
-
-
-
+|**Name** |**Description** |**Default** | **Properties**
+|`applicationInstanceLabelKey` |The `metadata.label` key name where Argo CD injects the app name as a tracking label.|`app.kubernetes.io/instance` |
+
+|`applicationSet`
+|`applicationSet` controller configuration options.
+| `_<object>_`
+a|* _<image>_ - The container image for the `applicationSet` controller. This overrides the `ARGOCD_APPLICATIONSET_IMAGE` environment variable.
```
Comment (skopacz1) [inline modules/gitops-argo-cd-properties.adoc:18]: Now that I think about it, is there a reason that the values in these lists, such as <image>, don't have backticks, but the values in the Default column (such as `_<object>_` in the preceding line) do?

**Review by skopacz1 — COMMENTED** [review]
Gave this another quick look-over, left a few more comments but otherwise this looks good to me!

File: modules/gitops-argo-cd-properties.adoc [hunk 4]
```diff
@@ -9,136 +9,117 @@
...
+|`configManagementPlugins`    |Add a configuration management plugin.| `__<empty>__` |
+
+|`controller`    |Argo CD Application Controller options.| `__<object>__`
+a|* `<processors.operation>` - The number of operation processors.
+  * `<processors.status>` - The number of status processors.
+  * `<resources>` - The container compute resources.
+  * `<logLevel>` - The log level used by the Argo CD Application Controller component. Valid options are `debug`, `info`, `error`, and `warn`.
+  * `<appSync>` - AppSync is used to control the sync frequency of Argo CD applications.
+  * `<sharding.enabled>` - Enable sharding on the Argo CD Application Controller component. This property is used to manage a large number of clusters to relieve memory pressure on the controller component.
+  * `<sharding.replicas>` - The number of replicas that are used to support sharding of the Argo CD Application Controller.
```
Comment (reginapizza) [inline modules/gitops-argo-cd-properties.adoc:34]: why was the rest of sharding taken out here?

File: modules/gitops-argo-cd-properties.adoc [hunk 5]
```diff
@@ -9,136 +9,121 @@
...
+
+|`sso` |Single Sign-on options.|`__<object>__`
+a|* `<image>` - The container image for Keycloak. This overrides the `ARGOCD_KEYCLOAK_IMAGE` environment variable.
+  * `<keycloak>` - Configuration options for Keycloak SSO provider.
+  * `<dex>` - Configuration options for Dex SSO provider.
+  * `<provider>` - The name of the provider used to configure Single Sign-on. Currently, the supported options are Dex and Keycloak.
+  * `<resources>` - The container compute resources.
+  * `<verifyTLS>` - Defines whether to enforce strict TLS checking when communicating with Keycloak service.
+  * `<version>` - The tag to use with the Keycloak container image.
+|`statusBadgeEnabled` |Enable application status badge.|`true` |
```
Comment (reginapizza) [inline modules/gitops-argo-cd-properties.adoc:117]: SSO is also different than what was there before

**Review by reginaelyse — APPROVED** [review]

**Pull request merged**